x/wasm/keeper/querier.go

```diff
@@ -335,3 +335,20 @@ func (q GrpcQuerier) InactiveContracts(c context.Context, req *lbmtypes.QueryIna
 		Pagination: pageRes,
 	}, nil
 }
+
+func (q GrpcQuerier) InactiveContract(c context.Context, req *lbmtypes.QueryInactiveContractRequest) (*lbmtypes.QueryInactiveContractResponse, error) {
```

Comment:
If there is no contract of the address provided, it will return false. In this case, the user cannot distinguish whether:

- the contract does not exist, OR
- the contract is active.

So, it would be helpful to throw an error if the contact does not exist.

Reply:
Do you want to add checking if the request address exists or not?
But the response is either true or false. So I just responsed false like balance even if there is no contract address.

Reply:
> So, it would be helpful to throw an error if the contact does not exist.

I agree this opinion.

Reply:
The x/bank does not complain about non-existing accounts. So it just returns a balance of zero, even if the query is for a non-existing account. It can also send coins to a non-existing account, creating the account in its logic.
However, x/wasm DOES complain about non-existing accounts, and it already has the corresponding error:
https://github.com/line/lbm-sdk/blob/2f72ab74f1b4059fa245c7d2146584357a2f3de4/x/wasm/keeper/keeper.go#L760
It's the reason why I thought it would be helpful to throw an error.